.evg.yml

```diff
@@ -405,16 +415,19 @@ functions:
             export MDBJDBC_VER=snapshot
           fi
 
-          case '${_platform}' in
+          case ${_platform} in
           ubuntu2204-64-jdk-8)
                JAVA_HOME=/opt/java/jdk8
           ;;
           ubuntu2204-64-jdk-11)
                JAVA_HOME=/opt/java/jdk11
           ;;
+          amazon2-arm64-jdk-11)
+               JAVA_HOME=/usr/lib/jvm/java-11
```

Comment:
There isn't a jdk8 installed on the host, only this one.  If we want 8 I can create a devprod ticket.

Reply:
I think we should be able to build with java 11, I'm pretty sure that I build with 13.